src/lightning_app/runners/cloud.py

```diff
@@ -98,8 +99,16 @@ def dispatch(
 
         print(f"The name of the app is: {app_config.name}")
 
-        work_reqs: List[V1Work] = []
         v1_env_vars = [V1EnvVar(name=k, value=v) for k, v in self.env_vars.items()]
+
+        if len(self.secrets.values()) > 0:
```

Comment:
no big deal but I wonder why we made `secrets` an object attribute instead of just passing it as an argument to `dispatch()` method. It looks odd to have to call `self.secrets` on the runtime object